include/ur_api.h

```diff
@@ -226,6 +226,8 @@ typedef enum ur_function_t {
     UR_FUNCTION_BINDLESS_IMAGES_IMPORT_EXTERNAL_MEMORY_EXP = 226,         ///< Enumerator for ::urBindlessImagesImportExternalMemoryExp
     UR_FUNCTION_BINDLESS_IMAGES_IMPORT_EXTERNAL_SEMAPHORE_EXP = 227,      ///< Enumerator for ::urBindlessImagesImportExternalSemaphoreExp
     UR_FUNCTION_ENQUEUE_NATIVE_COMMAND_EXP = 228,                         ///< Enumerator for ::urEnqueueNativeCommandExp
+    UR_FUNCTION_TENSOR_MAP_ENCODE_IM_2_COL_EXP = 231,                     ///< Enumerator for ::urTensorMapEncodeIm2ColExp
```

Comment:
Should this be 229?

Reply:
This is autogenerated so I don't think I have control over the number that is used to identify the entry point. Maybe @kbenzie can weigh in